script/test

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+trap cleanup 1 2 15
+
+cleanup() {
+  rm -rf "$temp_folder"
+}
+
+temp_folder="./test/tmp"
+fixtures_folder="./test/scss/fixtures"
+
+echo "\nGenerating CSS…\n"
+
+npx sass test/scss:$temp_folder --style=compressed --no-source-map
+npx sass scss/bitstyles.scss:$temp_folder/bitstyles.css --style=compressed --no-source-map
+npx postcss $temp_folder/*.css --replace
+npx prettier --write $temp_folder/*.css
+
+echo "\nComparing generated CSS against expected CSS…\n"
+
+diff -u -w --from-file $fixtures_folder/bitstyles-overrides.css $temp_folder/test-import-all.css \
+&& diff -u -w --from-file $fixtures_folder/bitstyles-overrides.css $temp_folder/test-use-all.css \
+&& diff -u -w --from-file $fixtures_folder/bitstyles-overrides.css $temp_folder/test-import-each.css \
+&& diff -u -w --from-file $fixtures_folder/bitstyles.css $temp_folder/bitstyles.css
+
+compare_result=$?
+
+cleanup
+exit $compare_result
```

Comment:
Wonder how actionable this is for developers 🤔

In order to know "what"  is off, I'd still have to run the internals of script myself, right? Maybe it makes sense to output the actual diff result as well.

Or am I missing something?

Reply:
The diff commands further up output any differences they find to stdout, so if there’s a problem it is actionable (as much as the diff output is readable. I tried to pick a good output format, but possibly YMMV)

Reply:
**TIL** "YMMV" 🤯 

https://www.howtogeek.com/693183/what-does-ymmv-mean-and-how-do-you-use-it/

Reply:
:D am I old, or is it because it’s English? 🙈